docs/source/dev/PlantUML.rst

```diff
@@ -8,22 +8,25 @@ For each UML diagram generated using PlantUML we also provide the source file ``
 
 Install PlantUML
 ----------------
-To install PlantUMl you need to
+To install PlantUMl you need to:
 
-- install a compatible java version
+- install a compatible ``java`` version
 - download ``plantUML.jar`` from the `PlantUML website <https://plantuml.com/download>`_ and place it in a folder of your choice.
```

Comment:
In general, why aren't we just linking to [this](https://plantuml.com/starting)?

Reply:
Even more general: [This page](https://plantuml.com/running) suggests that we could teach our docs to generate the images themselves. Might be worth an investigation.

Reply:
added a link and kept the instructions as as an TLDR:

Automatically generating UML-diagrams would be nice, but I would suggest to shift this to a future PR.